pkgs/development/python-modules/docutils/default.nix

```diff
@@ -17,12 +17,9 @@ let
     version = "0.21.2";
     pyproject = true;
 
-    disabled = pythonOlder "3.7";
-
-    src = fetchFromRepoOrCz {
-      repo = "docutils";
-      rev = "docutils-${version}";
-      hash = "sha256-Q+9yW+BYUEvPYV504368JsAoKKoaTZTeKh4tVeiNv5Y=";
+    src = fetchurl {
```

Comment:
Is this in conformance with the Note above?

Reply:
I suspect the note may already be out of date, or possibly mitigated by the fact that `fetchurl = fetchurlBool` during the bootstrap?

My change from `fetchFromRepoOrCz` to `fetchurl` should not cause problems, given that `fetchFromRepoOrCz` already uses `fetchurl` (via `fetchzip`).